playbooks/ORCHESTRATION.md

```diff
@@ -127,9 +127,70 @@ Manually check for:
 
 <!-- TODO: Add playbook for disaster/recovery steps. -->
 
-### Hardware Key Signing
+## Step 3: Add delegation (Optional)
+
+This step will add a delegated role to the top-level targets that is
+controlled by an external GitHub repository. Coordinate with the
+delegation keyholder to run the `add-delegation` (see
+https://github.com/sigstore/root-signing/blob/main/cmd/tuf/app/add-delegation.go).
+When creating the delegation with the command, a `target-meta`
+file has to be provided that lists the targets, similar to adding
+the top level targets.
+After the delegation metadata is added and signed, the delegation
+keyholder should open a PR against the ceremony branch.
+The name of the PR MUST be `feat/add-delegation for
+<delegation-name>`.
```

Comment:
Can we provide a snippet here?

Reply:
A lot of the PR generation in this repo is automated, I'm wondering if we can do the same here? Maybe an enhancement in a future PR?

Reply:
https://github.com/sigstore/root-signing/issues/751